TestProject1/Class2.cs

```diff
@@ -0,0 +1,9 @@
+﻿using System;
+
+namespace Task11
+{
+    public enum TransportType
+    {
+        plane, train, car, seaship, rivership
```

Comment:
Элементы перечисления принято называть с большой буквы.
```suggestion
        Plane, Train, Car, Seaship, Rvership
```